### PR TITLE
fix conflicts

### PR DESCRIPTION
**Pull request by rosemarymarano**
Backport of doc re-org changes, requested for the 1.4 release.

### CODE REVIEW

**Review by MelwynPaul — APPROVED** [review]
Thank you, Rosemary.
The TOC updates look good in 1.4 as well. 
One comment - I guess we should remove the 'Verrazzano Console' topic here as well.

**Comment by rosemarymarano** [comment]
@MelwynPaul The console page should have been removed in my previous commit. I'll check; I might have cut this branch before I merged it. If so, I'll include it in this commit. GOOD CATCH.

**Comment by rosemarymarano** [comment]
@MelwynPaul You were absolutely right! I guess the merge conflict didn't catch it because it looked like an add rather than a delete. In any case, thank you for keeping up with the details. I think that the PR is now good but I'll wait for the devs to check it.

**Review by MelwynPaul — APPROVED** [review]
Thank you, @rosemarymarano .
Yes, it was appearing in my local build (for this branch). 
I do not see the topic now (in the local build).

File: content/en/docs/setup/prereqs.md [hunk 1]
```diff
@@ -63,7 +63,12 @@ component, its version, and a brief description.
 | Keycloak                     | 15.0.2       | Provides single sign-on with Identity and Access Management.                             |
 | Kiali                        | 1.42.0       | Management console for the Istio service mesh.                                           |
 | kube-state-metrics           | 2.4.2        | Provides metrics about the state of Kubernetes API objects.                              |
+<<<<<<< HEAD
```
Comment (agv426) [inline content/en/docs/setup/prereqs.md:66]: Looks like a merge that wasn't resolve here

**Review by agv426 — APPROVED** [review]

**Pull request merged**